build.gradle

```diff
@@ -198,7 +198,7 @@ subprojects { subproject ->
 			postgresVersion = '9.1-901-1.jdbc4'
 			subethasmtpVersion = '1.2'
 			slf4jVersion = '1.7.11'
-			springIntegrationVersion = '5.0.0.M1'
+			springIntegrationVersion = '5.0.0.BUILD-SNAPSHOT'
 			springIntegrationKafkaVersion = '3.0.0.BUILD-SNAPSHOT'
 			springIntegrationSplunkVersion = '1.1.0.RELEASE'
 			springKafkaVersion = '1.1.2.BUILD-SNAPSHOT'
```

Comment:
👍 

Can you do these minor polishes on merge?

We can consider re-adding topic provisioning when we create the admin module.